### DISCUSSION TITLE
How to make a URL active in the text of the agent installer. For example, the URL to the personal data processing policy.

### DISCUSSION THREAD

**Discussion by Creogenesis** [Ylianst/MeshCentral#5767] [Q&A]
When inserting a URL such as https://help.service.com/policy, is it possible to make it active (so that the default browser opens upon clicking) to review the privacy and personal data processing policy?

If you simply enter the URL text, it remains just text.

              "installText": {
                "type": "string",
                "default": null,
                "description": "Text string to show in the agent installation dialog box."
                
Also interested in the possibility of increasing the font size of this text.

**Comment by si458** [comment]
Sadly it's only string as far as I'm aware
BUT
You could try `<a href='https://Google.com'>Google</a>` which is the html element for a hyperlink?

**Reply by Creogenesis** [reply]
Also interested in the possibility of increasing the font size of this text.

**Reply by Creogenesis** [reply]
`<a href='https://Google.com'>Google</a>`

I tried this option. It doesn't work.
